src/utils/order_validation_utils.ts

```diff
@@ -0,0 +1,89 @@
+import {ExchangeContractErrs, SignedOrder} from '../types';
+import {TokenWrapper} from '../contract_wrappers/token_wrapper';
+
+export class OrderValidationUtils {
+    public static async validateFillOrderBalancesAndAllowancesAndThrowIfInvalidAsync(
+        tokenWrapper: TokenWrapper, signedOrder: SignedOrder, fillTakerAmount: BigNumber.BigNumber,
```

Comment:
We are passing `TokenWrapper` into each call in this class. Why don't we instantiate the class with it, and store it as an instance variable?